src/System.CommandLine/Binding/ModelBinder{T}.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq.Expressions;
+
+namespace System.CommandLine.Binding
+{
+    public class ModelBinder<TModel> : ModelBinder
+    {
+        public ModelBinder() : base(ModelDescriptor.FromType<TModel>())
+        {
+        }
+
+        public void BindMemberFromOption<TValue>(
```

Comment:
We need to be able to add options and arguments without a generic class, or we force any code doing generalized AppModel stuff to use ugly reflection hacks. This could be provided a couple of ways, but creating an Add on BindingModel (not generic) would be nice.

Reply:
I'll add overloads for additional ways of configuring the bindings.